_sass/components/_identifier.scss

```diff
@@ -0,0 +1,134 @@
+.contact-bar {
+  padding-bottom: 0.5rem;
+  padding-top: 0.5rem;
+}
+
+.usa-identifier {
+  background-color: $theme-identifier-background-color;
+  color: $theme-identifier-color;
+  font-family: $font-sans;
+  font-weight: 400;
+  font-size: 1.5rem;
+  padding-top: 1rem;
+  padding-bottom: 3rem;
+
+  a {
+    color: $theme-identifier-color;
+  }
+
+  ul, ol {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+}
+
+.usa-identifier__required-links-list a {
+  font-size: 1.5rem;
+}
+
+a.usa-identifier__required-link {
+  color: $theme-identifier-color;
+
+  &:hover {
+   color: $theme-identifier-text-color-secondary;
+  }
+}
+
+.usa-identifier__container {
+  @media #{$breakpoint-lg} {
+    display: flex;
+    align-items: center;
+  }
+}
+
+.usa-identifier__logos {
+  padding-top: 1rem;
+  padding-right: 2rem;
+}
+
+.usa-identifier__logo {
+  display: block;
+  width: 5rem;
+}
+
+.usa-identifier__identity-disclaimer {
+  font-size: 1.6rem;
```

Comment:
Let's stick to the 1.5rem font-size mentioned in the component.

Reply:
This is now removed